custom-hooks/Exercise1/solution/src/Toggle.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from "react";
+
+interface ToggleProps {
+  on: boolean;
+  toggle: () => void;
+}
+const Toggle: FC<ToggleProps> = ({on, toggle}) => {
+
+
+  return (
+    <div>
+      <label className="toggle">
+        <input
+          onChange={toggle}
+          className="toggle-checkbox"
+          type="checkbox"
+          checked={on}
+        />
+        <div className="toggle-switch"></div>
+        <span className="toggle-label">{on ? "On" : "Off"}</span>
```

Comment:
I haven’t seen this UI so I’m guessing here, please ignore if I’m wrong: is this changing the label based on the status of the checkbox? If so, I would keep this label stable (maybe On or Enabled) so there’s a consistent label for the checkbox.